action-templates/actions/action-pr-labeler/action.yml

```
@@ -0,0 +1,26 @@
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout external repository
+      if: ${{ inputs.repository != '' }}
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      with:
+        repository: '${{ inputs.repository }}'
+    - name: Label PR
+      uses: actions/labeler@8558fd74291d67161a8a78ce36a881fa63b766a9 # v5
+      with:
+        configuration-path: ${{ inputs.configuration-path }}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)